zcash_client_backend/src/wallet.rs

```diff
@@ -54,7 +54,7 @@ pub struct WalletShieldedSpend {
 pub struct WalletShieldedOutput<N> {
     pub index: usize,
     pub cmu: bls12_381::Scalar,
-    pub epk: jubjub::ExtendedPoint,
+    pub ephemeral_key: [u8; 32],
```

Comment:
Would it make sense for this to be typed as `EphemeralKeyBytes`, which we're using elsewhere?